src/main/java/com/bina/varsim/util/StringUtilities.java

```diff
@@ -51,10 +52,39 @@ public static String concatenateArray(final ChrString[] a, final CharSequence de
         if (a == null) {
             return "";
         }
-        StringJoiner joiner = new StringJoiner(delimiter);
+        StringJoiner joiner = new StringJoiner(delimiter.toString());
         for (int i = 0; i < a.length; i++) {
             joiner.add(a[i].toString());
         }
         return joiner.toString();
     }
+
+    /**
+     * partial implementation of Java 8 StringJoiner
+     */
+    public static class StringJoiner {
```

Comment:
I see. Yeah, we have moved to Java 8, primarily due to HTSJDK being Java 8 now. Please update the manual. Thanks.